browser/src/RegionalGenomicConstraintTrack.tsx

```diff
@@ -0,0 +1,296 @@
+            {/* TODO:(rgrant) link to blog / help text when that is ready */}
+            {/* <InfoButton topic="regional-constraint" /> */}
+          </SidePanel>
+        )}
+      >
+        {({ scalePosition, width }: any) => (
+          <>
+            <TopPanel>
+              <Legend />
+            </TopPanel>
+            <PlotWrapper>
+              <svg height={height} width={width}>
+                <rect
+                  x={scalePosition(start)}
+                  y={30}
+                  width={scalePosition(stop) - scalePosition(start)}
+                  height={1}
+                  fill="#000"
+                />
+                {/* Conditional bar showing variant */}
+                {currentParams.variant && (
+                  <>
+                    <rect
+                      x={scalePosition((currentParams.variant as string).split('-')[1])}
+                      y={15}
+                      width={2}
+                      height={30}
+                      fill="#000"
+                    />
+                    <text
+                      x={scalePosition((currentParams.variant as string).split('-')[1])}
+                      y={12}
+                      dy="0.33rem"
+                      textAnchor="middle"
+                    >
+                      <Link to={`/variant/${currentParams.variant}`}>{currentParams.variant}</Link>
+                    </text>
+                  </>
+                )}
+                {regions.map((region: any) => {
+                  const startX = scalePosition(region.start)
+                  const stopX = scalePosition(region.stop)
+                  const regionWidth = stopX - startX
+
+                  return (
+                    <TooltipAnchor
+                      key={`${region.start}-${region.stop}`}
+                      // @ts-expect-error TS(2322) FIXME: Type '{ children: Element; key: string; region: an... Remove this comment to see the full error message
+                      region={region}
+                      tooltipComponent={RegionTooltip}
+                    >
+                      <g>
+                        <rect
+                          x={startX}
+                          y={22.5}
+                          width={regionWidth}
+                          height={15}
+                          fill={regionColor(region)}
+                          stroke="black"
+                        />
+                        {regionWidth > 30 && (
+                          <text x={(startX + stopX) / 2} y={30} dy="0.33em" textAnchor="middle">
+                            {region.z.toFixed(2)}
+                          </text>
+                        )}
+                      </g>
+                    </TooltipAnchor>
+                  )
+                })}
+              </svg>
+            </PlotWrapper>
+          </>
+        )}
+      </Track>
+    </Wrapper>
+  )
+}
+
+RegionalGenomicConstraintTrack.defaultProps = {
```

Comment:
`defaultProps` goes with class-based components: the only reason our functional components use it is because `ts-migrate` uses it in all cases. 

It's better instead to specify all the props in one type, and specify a default if wanted within the body of the component, e.g. `const {foo, bar, baz = 999} = props`.